benchmark/fluid/models/machine_translation.py

```diff
@@ -26,6 +26,10 @@
 import paddle.fluid.core as core
 import paddle.fluid.framework as framework
 from paddle.fluid.executor import Executor
+from models.model_base import get_decay_learning_rate
```

Comment:
model_base is not uploaded?

Reply:
Thanks for review, I added the benchmark/fluid/models/model_base.py file in next commit